### PR TITLE
Refactor: Consolidate import and usage of random

### PR DESCRIPTION
**Pull request by eumiro**

### CODE REVIEW

File: tests/models/test_cleartasks.py [hunk 1]
```diff
@@ -580,25 +581,23 @@ def test_dags_clear(self):
...
 
         DAG.clear_dags(dags, only_failed=True)
 
-        for i in range(num_of_dags):
-            tis[i].refresh_from_db()
-            if i != failed_dag_idx:
-                assert tis[i].state == State.SUCCESS
-                assert tis[i].try_number == 3
-                assert tis[i].max_tries == 1
+        for ti in tis:
```
Comment (uranusjr) [inline tests/models/test_cleartasks.py:591]: Wow this entire function can use a rewrite. (Not saying you should do it in this PR.)

**Review by uranusjr — APPROVED** [review]

**Review by Lee-W — COMMENTED** [review]
Same as https://github.com/apache/airflow/pull/34113. I'm wondering why we want to consolidate imports this way. Does that mean we should avoid `from ... import ...` pattern.

**Review by potiuk — APPROVED** [review]

**Comment by Lee-W** [comment]
> > Same as #34113. I'm wondering why we want to consolidate imports this way. Does that mean we should avoid `from ... import ...` pattern.
> 
> I'm suggesting having all imported modules/classes the same way, so in this case there's always `import random` and not `from random import random` which could confuse devs. `random` is a case where direct `import random` is better, since solitary `random()` or `randrange()` may be obvious, but `sample()` or `choice()` not.
> 
> There are cases where I do `from … import …`, e.g. `collections` or `pathlib`. The most important rule is to be repo-wide-consistent. I also usually do `import itertools as it`, but the maintainers suggested to keep it as `itertools`, which I do not object.

Yep, agree unifying them would be great 👍 Do we want to document it somewhere? (e.g., use `import random` instead of `from random import random`)

**Comment by potiuk** [comment]
I think those kind of rules are rarely read and we have two approaches there that 'work' 

a) hope that people will follow what is in the code (not perfect but mostly works) - this is a stage we are currently in


b) better - add pre-commit to do ast parsing and add rules of things we want do not want to have. I'd say it would be great if thos unification that can be done (imports are generally easy) are added as a rule to pre-commits 


Maybe you can work with @eumiro together to determine the list of rules that can be automatically verified and and write a pre&commit doing it ?

**Pull request merged**